src/client/components/app/Transactions/EnableSpigotTx.tsx

```diff
@@ -200,25 +237,64 @@ export const EnableSpigotTx: FC<EnableSpigotTxProps> = (props) => {
     );
   }
 
+  const renderRevFunc = () => {
+    if (revContractABI) {
+      return (
+        <TxFuncSelector
+          headerText="2. Rev Contract Funcs"
+          typeOptions={createListItems(selectedContractFunctions!)}
+          selectedType={funcType}
+        ></TxFuncSelector>
+      );
+    } else {
+      return (
+        <TxByteInput
+          headerText={t('components.transaction.enable-spigot.function-revenue')}
+          inputText={' '}
+          inputError={false}
+          byteCode={settingClaimFunc}
+          onByteCodeChange={handleClaimChange}
+          readOnly={false}
+        />
+      );
+    }
+  };
+
+  const renderTransferFunc = () => {
+    console.log(selectedContractFunctions);
+    if (revContractABI) {
+      return (
+        <TxFuncSelector
+          headerText="2. Transfer Contract Funcs"
```

Comment:
This needs to be hooked up to our public/locales/en. All user facing text needs to go through transaltion set up